test/AspNet.Security.OAuth.Providers.Tests/OAuthTests`1.cs

```diff
@@ -130,7 +132,10 @@ protected WebApplicationFactory<Program> CreateTestServer(Action<IServiceCollect
         /// The HTTP client to use for the remote identity provider.
         /// </returns>
         protected HttpClient CreateBackchannel(AuthenticationBuilder builder)
-            => builder.Services.BuildServiceProvider().GetRequiredService<IHttpClientFactory>().CreateClient();
+        {
+            using var serviceProvider = builder.Services.BuildServiceProvider();
```

Comment:
Why is a scope suddenly needed here?

Reply:
https://github.com/BADF00D/DisposableFixer suggested that there is indisposed object